great_expectations/datasource/fluent/interfaces.py

```diff
@@ -538,13 +547,13 @@ def test_connection(self, test_assets: bool = True) -> None:
             """One needs to implement "test_connection" on a Datasource subclass."""
         )
 
-    def _build_data_connector(self, data_asset_name: str, **kwargs) -> None:
+    def _build_data_connector(self, data_asset: _DataAssetT, **kwargs) -> None:
```

Comment:
@Kilo59 I checked, and am confirming that `Datasource. _build_data_connector()` interface definition should be deleted (its presence is a typographical error -- accidental left-over from an earlier iteration).  Even if a reason for this method may arise as part of a future architectural enhancement, it should be deleted for now.  Thank you.

Reply:
I'm using it now.